src/NuGetSdkResolver/NuGet.MSBuildSdkResolver.csproj

```diff
@@ -64,6 +64,9 @@
     <Compile Include="..\Shared\FileUtilities.cs">
       <Link>Shared\FileUtilities.cs</Link>
     </Compile>
+    <Compile Include="..\Shared\TempFileUtilities.cs">
+      <Link>Shared\TempFileUtilities.cs</Link>
```

Comment:
Is this needed? I don't see any changes in the project that could use it.

Reply:
The `TestEnvironment` that is compiled into the unit test library needs to be able to call some methods in `TempFileUtilities`. I couldn't get things to compile when I included it into the unit test library so I put it in the SDK resolver like it is in other libraries